app/src/WebsiteGenerator/TemplateRenderer/TemplateRenderer.php

```diff
@@ -59,6 +69,8 @@ public function render(string $filename, string $templateName, array $data)
 
         $data['page']['template'] = $templateName;
 
+        $this->twig->addGlobal('info', $this->websiteInfoRepository->getWebsiteInfo());
```

Comment:
This is a question more than a comment. 

What does the addGlobal mean and how is it different to just adding the the array we hand over to twig?

Reply:
In this case, it's no different. But it does mean that the info can be used in other places where it's needed like in base.html.twig and the navigation and footer partials.

Reply:
Great. In that case yes I think that's exactly what we should be doing.